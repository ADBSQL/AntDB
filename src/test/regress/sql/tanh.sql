set grammar to oracle;
select tanh(exp(2)) from dual;
select tanh(-1) from dual;
select tanh('0.9') from dual;
select tanh(0) from dual;
select tanh(to_char(0.5)) from dual;
select tanh('0.9') from dual;
select tanh(null) from dual;
select tanh('') from dual;
