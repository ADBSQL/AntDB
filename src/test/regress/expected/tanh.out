set grammar to oracle;
select tanh(exp(2)) from dual;
       tanh        
-------------------
 0.999999236204229
(1 row)

select tanh(-1) from dual;
        tanh        
--------------------
 -0.761594155955765
(1 row)

select tanh('0.9') from dual;
       tanh        
-------------------
 0.716297870199024
(1 row)

select tanh(0) from dual;
 tanh 
------
    0
(1 row)

select tanh(to_char(0.5)) from dual;
       tanh       
------------------
 0.46211715726001
(1 row)

select tanh('0.9') from dual;
       tanh        
-------------------
 0.716297870199024
(1 row)

select tanh(null) from dual;
 tanh 
------
     
(1 row)

select tanh('') from dual;
 tanh 
------
     
(1 row)

